src/makeInfinityAdder.js

```diff
@@ -1,10 +1,35 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let clearCall = 0;
```

Comment:
The variable 'clearCall' is used to track if the function was called without arguments. However, this approach requires two calls without arguments to reset the 'result'. Consider simplifying the logic so that a single call without arguments both returns the current sum and resets the 'result'.